app/views/users/sessions/verify_2fa.html.erb

```diff
@@ -1,4 +1,11 @@
 <h1>Two-factor authentication</h1>
+
+<script>
+  window.onload = function() {
+      document.getElementById("code").focus();
```

Comment:
Let's do optional chaining or an `if` here: we do not want to hard fail if the element is not found (it is guaranteed now, but future-proofing it is beneficial). Either `document.getElementById("code")?.focus()` or (arguably better as optional chaining support is a bit lower still):

```javascript
const codeElem = document.getElementById("code");

if (codeElem) {
    codeElem.focus();
}
```